build.gradle

```diff
@@ -97,6 +100,7 @@ dependencies {
     testImplementation "org.junit.jupiter:junit-jupiter-engine"
     testImplementation "org.hamcrest:hamcrest:2.2"
     testImplementation "org.hamcrest:hamcrest-library:2.2"
+    testImplementation "bad.robot:simple-excel:1.2"
```

Comment:
Seems not used

Reply:
Uses for comparing original excel file with converted: ```WorkbookMatcher.sameWorkbook(expected)```